velox/exec/HashBuild.h

```diff
@@ -84,10 +84,23 @@ class HashBuild final : public Operator {
   void close() override {}
 
  private:
+  // Invoked to setup hash table to build.
+  void setupTable();
+
   void addRuntimeStats();
 
+  const std::shared_ptr<const core::HashJoinNode> joinNode_;
+
   const core::JoinType joinType_;
 
+  const std::shared_ptr<HashJoinBridge> joinBride_;
```

Comment:
I can't find how joinBride_ is used. Would you point me to relevant code?

Reply:
Updated. It will be used in disk spilling control in followup as well. Thanks!